src/components/organisms/news/news.scss

```
@@ -0,0 +1,31 @@
+@use "../../variables" as *;
+
+.news {
+  display: flex;
+  flex-direction: column;
+  .news-article {
+    background-color: $white;
+    box-sizing: border-box;
+    border: 1px solid $gray_gainsboro;
+    height: 85px;
+    padding: 20px;
+    .time {
+      font-size: 12px;
+      line-height: 12px;
```

```suggestion
      line-height: 1;
```
これでも同じ意味になります。

> <number> (単位なし)
使用値は、この単位のない [<number>](https://developer.mozilla.org/ja/docs/Web/CSS/number) に要素のフォントサイズを掛けたものになります。計算値は、指定された <number> と同じです。ほとんどの場合、継承時の予期しない結果を避けるために、これが line-height を設定する好ましい方法です。

https://developer.mozilla.org/ja/docs/Web/CSS/line-height
好ましい方法とされているので、知っておいて損はないと思います。